packages/ethereum/web3-ethereum/src/index.ts

```diff
@@ -150,7 +130,7 @@ export class Web3Ethereum implements EthereumProvider.Ethereum {
 
 	async getChainId(): Promise<number> {
 		try {
-		  return +(await this.config.web3.eth.getChainId(NumberDataFormat))
+		  return +(await this.config.web3.eth.getChainId())
```

Comment:
I would recommend: 
```
const chainIdRaw = await this.config.web3.eth.getChainId()
return toBn(chainIdRaw).toNumber()
```

It will parse hexadecimal and non-hexadecimal responses